overlord/devicestate/handlers.go

```diff
@@ -42,11 +43,18 @@ func (m *DeviceManager) doMarkPreseeded(t *state.Task, _ *tomb.Tomb) error {
 		return err
 	}
 
+	systemKey, err := interfaces.RecordedSystemKey()
+	if err != nil {
+		return fmt.Errorf("internal error: cannot get recorded system key: %v", err)
```

Comment:
given this is reading from disk, not clear it should be marked as "internal error"